samanthas_telegram_bot/conversation/callbacks/registration/student.py

```diff
@@ -54,9 +54,17 @@ async def ask_communication_language_or_start_assessment_depending_on_learning_e
     query, data = await answer_callback_query_and_get_data(update)
 
     if data == "year_or_more":
+        logger.info(
+            f"Chat {update.effective_chat.id}. "
+            f"Student has learned English for year or more. Starting assessment"
+        )
         await prepare_assessment(context, query)
         return ConversationState.ASK_ASSESSMENT_QUESTION
 
+    logger.info(
+        f"Chat {update.effective_chat.id}. "
```

Comment:
Maybe switch-case or some map of `level:log_text`?

Reply:
Not sure... Firstly, I do a `return` right after `if`, which prevents spaghetti. Secondly, the state returned is different depending on `data`, so it's not just about what I write to the log.

Reply:
Ok, agree